generated/gulpfile.js

```diff
@@ -55,12 +55,16 @@ gulp.task('buildJS', ['lintJS'], function () {
 
 gulp.task('testServerJS', function () {
     require('babel-register');
+    //testing environment variable 
+    process.env.NODE_ENV = 'testing';
 	return gulp.src('./tests/server/**/*.js', {
 		read: false
 	}).pipe(mocha({ reporter: 'spec' }));
 });
 
 gulp.task('testServerJSWithCoverage', function (done) {
+    //testing environment variable 
+    process.env.NODE_ENV = 'testing';
```

Comment:
@omriBernstein @joedotjs I am not exactly certain the environment wanted in this situation, but I would think you would want it here as well as tests seem to be run in the same way